README.md

```diff
@@ -1009,6 +1009,13 @@ export interface TTFBAttribution {
    * processing time.
    */
   requestDuration: number;
+  /**
+   * The total time from the first byte of the response was received. Until the
+   * first byte of the document was received. This will only be non-zero for
+   * servers using Early Hints and where browsers support sending this additional
+   * timing. This time is after the TTFB metric.value.
+   */
+  documentDuration: number;
```

Comment:
isn't that the first byte of the headers of the document?

Reply:
It's a duration. So the time between EH first bytes and document first bytes.

But yes it's basically:
`finalResponseHeadersStart` - `firstInterimResponseStart`
(with some extra handling for Chrome 115 - 132)

Reply:
so it's the time between EH first byte and the document's *headers* first byte, right?

Reply:
Correct. `documentHeadersDuration` or `documentResponseDuration` was a little long winded. But I'm up for a better name if we have it!

Reply:
I think the name is fine, but the docs may have room for clarifications

Reply:
WDYT:

```
  /**
   * The total time, after TTFB, after which the document started to be
   * received. This will only be non-zero for servers using Early Hints and
   * where browsers support sending this additional timing. This is the time
   * between the Early Hints first response (TTFB) and when the actual document
   * response started with its HTTP headers.
   */
```

Reply:
I'd go with "document's HTTP headers" in the first line, instead of "document". Otherwise, looks great!